packages/dev/docs/src/VersionBadge.js

```diff
@@ -11,17 +11,25 @@
  */
 
 
-import badgeStyles from '@adobe/spectrum-css-temp/components/label/vars.css';
+import {Badge} from '@react-spectrum/badge';
```

Comment:
this will need to be temporarily pulled out so we can merge. otherwise, the production docs will try to find this package but it won't be published yet.